### DISCUSSION TITLE
Отображение прогноза

### DISCUSSION THREAD

**Discussion by Ogurtsov-D** [IATkachenko/HA-YandexWeather#45] [Ideas]
При включении отображения прогноза на карточке HA выглядит вот так: 
![image](https://user-images.githubusercontent.com/78870757/181014288-8eb3bcb9-d882-4b6d-a939-f0b308c0421f.png)
Я понимаю, что тариф "погода на Вашем сайте" дает прогноз всего на 2 дня, но внешний вид прогноза очень уж аляпистый получается. Тут места на 5 дней прогноза.

Вопрос: Может еще 3 дня можно откуда-то еще подтянуть?

**Comment by IATkachenko** [comment]
Он даже не на "дня", а на "временных интервала" (утро, день, вечер, ночь), которые так мапятся (соответственно "вечер" попал на вторник, а "ночь" уже на среду. Панель разработчика -> состояния видны исходные timestamp).

Дополнительные прогнозные данные, к сожалению, взять неоткуда.
Можно каждый прогнозный интервал удваивать или утраивать. На карточке тогда будет меньше пустого места, но будет вводить в заблуждение.

**Reply by dadmetal** [reply]
Здравствуйте! Тоже интересно. А почему бы с сайта не парсить значения?
Есть php-скрипт, могу посодействовать, нужно будет только долготу и широту проставить.
Может аналог на python накидаете.

**Reply by IATkachenko** [reply]
Парсить сайт -- не то чтобы надежное решение... Тогда уж iFrame в интерфейс добавить проще.

**Reply by dadmetal** [reply]
За последние три года верстка не изменилась. С фрейма не выцепить потом значение в HA.
